libraries/FS/src/vfs_api.cpp

```diff
@@ -540,6 +540,31 @@ String VFSFileImpl::getNextFileName()
     return name;
 }
 
+String VFSFileImpl::getNextFileName(bool *isDir)
+{
+    if (!_isDirectory || !_d) {
+        return "";
+    }
+    struct dirent *file = readdir(_d);
+    if (file == NULL) {
+        return "";
+    }
+    if (file->d_type != DT_REG && file->d_type != DT_DIR) {
+        return "";
+    }
+    String fname = String(file->d_name);
+    String name = String(_path);
+    if (!fname.startsWith("/") && !name.endsWith("/")) {
+        name += "/";
+    }
+    name += fname;
+	// check entry is a directory
+	if(isDir) {
+		*isDir = (file->d_type == DT_DIR);
+	}
```

Comment:
That check is good for sure. Can you just remove the spaces before the part so its aligned with other code?